src/app/components/PortraitVideoCarousel/index.tsx

```diff
@@ -125,6 +164,24 @@ const PortraitVideoCarousel = ({
           </div>
         </div>
       </div>
+
+      {isModalOpen && selectedItem && (
+        <PortraitVideoModal
```

Comment:
I'd recommend using [createPortal](https://react.dev/reference/react-dom/createPortal) here to render the modal outside of our React root element and place it somewhere inside the the `body`. This should allow us to properly style it without too many `z-index` weirdness.

There is an example within the documentation around using Portal for modal purposes: https://react.dev/reference/react-dom/createPortal#rendering-a-modal-dialog-with-a-portal

Reply:
See https://github.com/bbc/simorgh/pull/12700/commits/3d753803a2621ee110f9644f321e46139d8aee6f